src/components/cards/experience-listing-card/ExperienceListingCard.vue

```diff
@@ -0,0 +1,221 @@
+<template>
+  <div class="experience-listing-card">
+    <div class="container">
+      <div class="image">
+        <img :src="image" />
+      </div>
+
+      <div class="body">
+        <p
+          :class="['status', isListed ? 'isListed' : '']"
+          v-if="deviceSize <= deviceSizes.s"
+        >{{ checkListed }}</p>
+        <p class="title">{{ title }}</p>
+        <p class="country" v-if="deviceSize > deviceSizes.s">{{ country }}</p>
+        <div class="actions" v-if="deviceSize <= deviceSizes.s">
+          <div>
+            <a>{{ listingAction }}</a>
+          </div>
+          <template>
+            &nbsp;&bull;&nbsp;
```

Comment:
Why don't use style to define some spacing ?

Reply:
Done